functorch/test/test_pythonkey.py

```diff
@@ -547,6 +547,41 @@ def forward(self, x, y):
         assert torch.allclose(inputs[0].grad, cloned_inputs[0].grad)
         assert torch.allclose(inputs[1].grad, cloned_inputs[1].grad)
 
+    def test_aot_module_simplified_preserves_stack_trace(self):
+
+        class MockModule(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.linear = torch.nn.Linear(20, 30)
+
+            def forward(self, x, y):
+                return (self.linear(x) + y, )
+
+        tracer = torch.fx.Tracer()
+        tracer.record_stack_traces = True
+        graph = tracer.trace(MockModule())
+        mod = torch.fx.GraphModule(tracer.root, graph)
```

Comment:
This call pattern doesn't look very realistic; aren't most people going to be calling `make_fx` in PT2?

Reply:
Yes, I know... this is the easiest way to populate the stack trace for a graph module ... 

make_fx is another entry point where graph module can get created... Do we want to introduce a flag to enable record_stacktrace for pythonkey_tracer within make_fx?